projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -134,16 +134,40 @@ def _landmark2numpy(landmarks):
 
 
 if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input_image_topic", help="Topic name for input image",
+                        type=str, default="/usb_cam/image_raw")
+    parser.add_argument("-o", "--output_category_topic", help="Topic name for output recognized category",
+                        type=str, default="/opendr/landmark_expression_recognition")
+    parser.add_argument("-d", "--output_category_description_topic", help="Topic name for category description",
+                        type=str, default="/opendr/landmark_expression_recognition_description")
```

Comment:
This is a new addition in #306, which i didn't mention in the tracker issue (will edit it in). It serves to handle the `None` values passed through argparse as string type instead of none type.

```suggestion
    parser.add_argument("-o", "--output_category_topic", help="Topic name for output recognized category",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/landmark_expression_recognition")
    parser.add_argument("-d", "--output_category_description_topic", help="Topic name for category description",
                        type=lambda value: value if value.lower() != "none" else None, 
                        default="/opendr/landmark_expression_recognition_description")
```